DasharoPayloadPkg/BlSupportPei/BlSupportPei.c

```diff
@@ -604,7 +610,7 @@ BlPeiEntryPoint (
   //
   // Set cache on the physical memory
   //
-  MtrrSetMemoryAttribute (BASE_1MB, LowMemorySize - BASE_1MB, CacheWriteBack);
+  MtrrSetMemoryAttribute (BASE_1MB, PldMemInfo.SystemLowMemTop - BASE_1MB, CacheWriteBack);

```

Comment:
Is commit message correct on this?  This is the top of the highest reserved memory region below 4G, not the highest region.  Also, the way it's obtained looks weird:
```c
SystemLowMemTop = ((UINT32)(Base + Size) + 0x0FFFFFFF) & 0xF0000000
```
```
(0xFED45000 + 0x0fffffff) & 0xf0000000 =  0x10ED44FFF & 0xf0000000 = 0
```

Reply:
Yes, it didn't make sense, I don't really understand what that value was supposed to do. 256MB mask suggests it may be MMCONF, but it isn't fixed relatively to other MMIO. Reworked to actually check for MMIO.

https://github.com/Dasharo/edk2/compare/b67eb08260c40f463eca88298e548cc70f8d3831..97938d04f1df2e564c1986fd34dc0dd00467cf54#diff-e11426c5786175fdf345eec361e1c716ab6792d8fddbd5b005e02ab589ad3d7bL373-R383